server/db/repositories/helpers.go

```diff
@@ -133,12 +136,22 @@ func compileQuery(collectionName string, q dal.Query) dal.CompiledQuery {
 		}
 	}
 
-	if q.Pagination.Page > 0 {
+	if q.Pagination.Cursor.IsEmpty() == false {
```

Comment:
simplify `!q.Pagination.Cursor.IsEmpty()`?

Reply:
Readability? :)

Reply:
It's more habitual form

Reply:
@golangcibot + @3timeslazy > @ziflex